Altis_Life.Altis/core/actions/fn_pickupItem.sqf

```diff
@@ -7,13 +7,13 @@
     Description:
     Master handling for picking up an item.
 */
-private ["_itemInfo","_itemName","_illegal","_diff"];
 if ((time - life_action_delay) < 2) exitWith {hint localize "STR_NOTF_ActionDelay"; INUSE(_this);};
 if (isNull _this || {player distance _this > 3}) exitWith {INUSE(_this);};
 
-_itemInfo = _this getVariable ["item",[]]; if (count _itemInfo isEqualTo 0) exitWith {deleteVehicle _this;};
-_illegal = ITEM_ILLEGAL(_itemInfo select 0);
-_itemName = ITEM_NAME(_itemInfo select 0);
+private _itemInfo = _this getVariable ["item",[]]; 
+if (count _itemInfo isEqualTo 0) exitWith {deleteVehicle _this;};
+private _illegal = ITEM_ILLEGAL(_itemInfo select 0);
```

Comment:
Might as well be worth storing in a variable for readability, ``_itemInfo params [["_whateverItIs", type, [type]]``